templates/demographics_survey_form.html

```diff
@@ -56,7 +80,7 @@
             <label class="label">Please select your birth year.</label>
             <div class="control">
                 <div class="select">
-                    <select title="Select options for eliciting birth year." name="birthyear" required="requried">
```

Comment:
What about just `<select title="..." name="..." required >` which I think is actually valid syntax.